packages/directory/app/styles/navi-directory/components/dir-table-filter.less

```diff
@@ -12,14 +12,19 @@
     border-radius: 5px;
     color: @navi-gray-700;
     cursor: pointer;
-    display: grid;
     font-family: @font-family-thin;
-    grid-template-columns: 1fr 5px;
     margin-left: auto;
     padding: 5px 10px;
     text-align: left;
     user-select: none;
     width: 120px;
+    display: flex;
```

Comment:
Looks good, but we try to keep our styles in alphabetical order. Could you move display and justify-content up?

Reply:
Yes sure. I will keep this in mind.